### PR TITLE
Fix NAN result while submitting empty time & update prefilled date and time for all form (India Time)

### PR DESCRIPTION
**Pull request by rahuldhangar**
Fix NAN result while submitting empty time
update prefilled date and time for all form (India Time)

PS: Kindly add "hacktoberfest-accepted" tag to this PR before merging it. Thank you!

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -17,6 +17,25 @@ const form = document.querySelector(".diff-form");
 const result = document.querySelector(".result");
 const popContainer = document.querySelector(".pop-container");
 
+//adding event listener to initialize Date when page is loaded
+// Todo: Below Event Listener can be replaced with jQuery document.ready() but require jQuery lib
+document.addEventListener('DOMContentLoaded', function() {
+    // Get current date in GMT
+    var d = new Date();
+    var nd = d.toLocaleString('en-US', { timeZone: 'America/New_York' });
```
Comment (amitoj-singh) [inline src/app.js]: Is this `nd` required? I think it can be removed safely.

**Review by amitoj-singh — CHANGES_REQUESTED** [review]
please remove `nd` as this is not required.

File: src/app.js (hunk 1 above)
Comment (rahuldhangar) [reply src/app.js]: Sorry, my bad! Line 25 is unrequired as the variable "nd" is never used (I created it while testing but forgot to remove it).
```suggestion
```

**Review by amitoj-singh — APPROVED** [review]

**Comment by rahuldhangar** [comment]
Thanks for approving the changes. You can now merge this with master if all looks good 👍

**Pull request merged**